doc/Manual.md

```diff
@@ -218,6 +241,14 @@ restore the contents of the latest snapshot to `/tmp/restore-work`:
     enter password for repository:
     restoring <Snapshot of [/home/user/work] at 2015-05-08 21:40:19.884408621 +0200 CEST> to /tmp/restore-work
 
+Use the word 'latest' to restore the last backup. You can also combine 'latest'
+with the --host and --path filters to choose the last backup for a specific
```

Comment:
Use backticks around verbatim strings like `latest` and command-line parameters like this:
```
`latest`

`--host` and `--path`
```